pom-step-02.xml

```diff
@@ -186,13 +186,14 @@
                 <executions>
                     <execution>
                         <id>package-jar-with-dependencies</id>
-                        <phase>prepare-package</phase>
```

Comment:
Why change the phase?

Reply:
This helps the attaching not to temporarily treat it as a  temporary build output